docker-compose.example.yml

```diff
@@ -1,28 +1,38 @@
-version: "3.7"
+version: "2.4"
 services:
   db:
     image: postgres:11 # pin postgres to a major version
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddydb
     environment:
       - PGDATA=/db-data
       - POSTGRES_PASSWORD=postgres # has to correspond with POSTGRES_PASSWORD in APP
     volumes:
       - db:/db-data:rw
+    restart: unless-stopped
   app:
     image: babybuddy/babybuddy
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddy
```

Comment:
In keeping the style of the `db` service, lets name this `babybuddyapp`.

Reply:
Agreed, updated!